### ISSUE TITLE
Is it normal for parser-125k Multicore score lower than Singlecore score?

### ISSUE THREAD

**Issue by bongo01** [eembc/coremark-pro#12]
<img width="454" alt="CoreMark Pro Hynix" src="https://user-images.githubusercontent.com/108515950/176849857-5739465a-42ad-4b23-aae5-ddc2ae0f6fc8.png">

I run CoreMark Pro on Cygwin in Windows 11.
When testing parser-125k multicore, I see that it didn't use all CPU threads like the other testing.

**Comment by petertorelli** [comment]
That's highly unusual. Do you observer the same poor scaling every run? Are each of the tests running for at least 10 seconds?

**Comment by bongo01** [comment]
Yes, I observe the same poor scaling every run (around 4-5 runs) and also each  of the tests running for at least 10 second. My CPU is AMD Ryzen 5800H

**Comment by bongo01** [comment]
Ok. I found that parser-125k is normal when I run on Ubantu in VirtualBox. 
I ran "make TARGET=linux64 certify-all XCMD='-c16'" in both Cygwin and Ubantu. 
I can't run "make TARGET=cygwin certify-all XCMD='-c16'" because there is an error "cygwin.mak : No such file or directory".

**Comment by petertorelli** [comment]
Glad you solved the problem in Ubuntu. But running under a VM is not a good idea as you are subject to pre-emption at a higher level by the host OS.

How were you able to run under cygwin in your first comment, but cannot run under it now?
